cogs/fun.py

```diff
@@ -4,11 +4,102 @@
+                      description="Get a random fact about cats!",
+                      syntax=None,
+                      examples=[argument("", "Returns a interesting fact about cats")])
+    async def catfact(self, ctx):
+        link = "https://catfact.ninja/fact?max_length=300/"
+
+        fact = await self.aiohttp_request(link, headers=None)
+        fact = json.loads(fact)
+        embed = discord.Embed(title="Cat Facts", color=misc.random_color(),
+                              description=fact["fact"])
+        await ctx.send(embed=embed)
+
+    @commands.command(aliases=["dogfacts"], cls=Command,
+                      description="Get a random fact about dogs!",
+                      syntax=None,
+                      examples=[argument(None, "Returns a interesting fact about dogs")])
+    async def dogfact(self, ctx):
+        link = "http://dog-api.kinduff.com/api/facts?numbers=1"
+        fact = await self.aiohttp_request(link, headers=None)
+        fact = json.loads(fact)
+        embed = discord.Embed(title="Dog Facts", color=misc.random_color(),
+                              description=fact["facts"][0])
+        await ctx.send(embed=embed)
+
+    @commands.command(aliases=["dogpic", "dog"], cls=Command,
+                      description="Get a random picture of a dog",
+                      syntax=None, examples=[argument("", "Get a cute picture of a dog")])
+    async def dogpics(self, ctx):
+        link = "https://dog.ceo/api/breeds/image/random/Fetch"
+        picture_data = await self.aiohttp_request(link, headers=None)
+        picture_data = json.loads(picture_data)
+
+        embed = discord.Embed(title="Here's a picture of a dog!", color=misc.random_color())
+        embed.set_image(url=picture_data["message"][0])
+        await ctx.send(embed=embed)
+
+    @commands.command(aliases=["catpic", "cat"], cls=Command,
+                      description="Get a random picture of a cat!",
+                      syntax=None, examples=[argument("", "Get a cute picture of a cat")])
+    async def catpics(self, ctx):
+        link = "https://api.thecatapi.com/v1/images/search?format=json?size=med"
+        picture_data = await self.aiohttp_request(link, headers=None)
+        picture_data = json.loads(picture_data)
+
+        embed = discord.Embed(title="Here's a picture of a cat!", color=misc.random_color())
+        embed.set_image(url=picture_data[0]["url"])
+        await ctx.send(embed=embed)
+
+    @commands.command(aliases=["choose", "choice"], cls=Command,
+                      description="Chooses a random item out of a list for you!",
+                      syntax=[(True, "List of items")],
+                      usage_information=[usage_info("", "List of items must be separated by commas or spaces")],
+                      examples=[example("1,2,3,4", "Returns a random selection like 2")])
+    async def random(self, ctx, *, message):
+        """Chooses a random item out of a list for you!
```

Comment:
D205: 1 blank line required between summary line and description (found à)
D400: First line should end with a period (not '!')
First line of dosctring should be in imperative mood